apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -26,7 +33,12 @@ interface MockServerInterface {
   fun takeRequest(): MockRecordedRequest
 }
 
+abstract class BaseMockServer(override val mockDispatcher: MockDispatcher) : MockServerInterface {
+  override fun enqueue(mockResponse: MockResponse) {
+    (mockDispatcher as QueueMockDispatcher).enqueue(mockResponse)
+  }
+}
```

Comment:
Nitpicking: I understand this is used to factor in the `enqueue` between all platforms? I wouldn't mind inlining it to avoid the abstract class.

Reply:
👍 done in 313ef8bffbd82dd309f20e7713f04f1e5a395684